docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,132 @@
+# ScheduledJob Controller
```

Comment:
The problem with "Cron" is that is technically focused (a particular linux
tool) but is opaque to higher level users who come from different
backgrounds.

Even the Chronos GitHub page describes it with "It allows you to schedule
your jobs using ISO8601 repeating interval notation, which enables more
flexibility in job scheduling".  RepeatingJob or CalendarJob or
RecurringJob all seem less clear to an outside user (someone who is not an
expert at these systems) than ScheduledJob.

On Fri, Jul 31, 2015 at 12:41 PM, Timothy St. Clair <
notifications@github.com> wrote:

> In docs/proposals/scheduledjob.md
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11980#discussion_r35991090>
> :
>
> > @@ -0,0 +1,132 @@
> > +# ScheduledJob Controller
>
> Comment still holds on name
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11980/files#r35991090>
> .
>



-- 
Clayton Coleman | Lead Engineer, OpenShift